README.md

```diff
@@ -274,6 +274,11 @@ It's possible to translate query string parameters by listening to the `translat
 
 For a possible implementation of the `YourModel::translateParams` method look at the example under `URL parameter translation` from above.
 
+## Extend theme scan
+      Event::listen('rainlab.translate.afterScan', function (ThemeScanner $scanner) {
```

Comment:
Event name still needs to be changed here. Please also add a newline between the header and the code example.